### DISCUSSION TITLE
Support $xdg_config_home for user's main config file (~/.wp-cli/config.yml)

### DISCUSSION THREAD

**Discussion by ZuBB** [wp-cli/wp-cli#5999] [Ideas]
Basically would be nice if `wp-cli` check also `$xdg_config_home` dir for its config file (in default settings it will be `~/.config/wp-cli/config.yml`).

Motivation: `wp-cli` throws ANSI esc sequences in the terminal. so I need to set `no-color` globally. But I also like my `~/` to be uncluttered.

Some refs
* https://github.com/wp-cli/wp-cli/issues/5833#issuecomment-2386575668
* https://github.com/wp-cli/wp-cli/issues/1747

**Comment by swissspidy** [comment]
Created an issue for this feature request: https://github.com/wp-cli/wp-cli/issues/6053
